apps/server/src/modules/tldraw/utils/utils.ts

```diff
@@ -0,0 +1,256 @@
+import { setInterval, clearInterval } from 'timers';
+import { Doc } from 'yjs';
+import { Awareness, encodeAwarenessUpdate, removeAwarenessStates, applyAwarenessUpdate } from 'y-protocols/awareness';
+import { encoding, decoding, map } from 'lib0';
+import { writeUpdate, readSyncMessage, writeSyncStep1 } from 'y-protocols/sync';
+import { Configuration } from '@hpi-schul-cloud/commons';
```

Comment:
This is not the way how we want to handle configuration in nestjs. This is the legacy way.

Reply:
Could you please send us how this shoul look like in non-legacy way?

Reply:
This point and the structure of the file is still open. For me it look like a service and not a util. If you do it in this way, than the configuration can also handled different and easier.